src/frontend/screens/PresetChooser.tsx

```diff
@@ -89,16 +88,18 @@ export const PresetChooser: NativeNavigationComponent<'PresetChooser'> = ({
   }, [navigation, existingPreset, handleGoBack]);
 
   const presetsList = !presets
-    ? null
+    ? []
     : Array.from(presets)
         // Only show presets where the geometry property includes "point"
         .filter(p => p.geometry.includes('point'))
         // Sort presets by sort property and then by name, then filter only point presets
-        .sort((a, b) => {
-          return a.name.toLowerCase().localeCompare(b.name.toLowerCase());
-        });
+        .sort((a, b) =>
+          a.name.toLowerCase().localeCompare(b.name.toLowerCase()),
+        );
```

Comment:
the ternary is not actually necessary here (I think i originally added it...)

presets will always be an array (sometimes it can be empty, but still an array regardless)

you can just do 

```ts
const presetsList = Array.from(presets)
        // Only show presets where the geometry property includes "point"
        .filter(p => p.geometry.includes('point'))
        // Sort presets by sort property and then by name, then filter only point presets
        .sort((a, b) =>
          a.name.toLowerCase().localeCompare(b.name.toLowerCase()),

```

Reply:
Ok fixed. Thanks for the tip.